pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -856,8 +858,13 @@ default void delete(String topic, boolean force) throws PulsarAdminException {
      * @throws PulsarAdminException
      *             Unexpected error
      */
-    TopicStats getStats(String topic, boolean getPreciseBacklog,
-                        boolean subscriptionBacklogSize) throws PulsarAdminException;
+    TopicStats getStats(String topic, boolean getPreciseBacklog, boolean subscriptionBacklogSize,
```

Comment:
We can't change the API directly, this will introduce a compatibility issue while users upgrade to the new version.

Reply:
We can introduce a new API TopicStats getStats(String topic, GetStatsOptions options);
So can we can introduce more params without breaking the API.

Reply:
Maybe the method below (Line 864 - 866) can resolve the compatibility issue because it's the original API with the `getEarliestTimeInBacklog` default to be `false`?

Reply:
No, all of the getStats methods is a public API, users might be used in their application, if we change one of them, users will encounter compatibility issues. 

```java
    TopicStats getStats(String topic, boolean getPreciseBacklog,
                        boolean subscriptionBacklogSize) throws PulsarAdminException;

    default TopicStats getStats(String topic, boolean getPreciseBacklog) throws PulsarAdminException {
        return getStats(topic, getPreciseBacklog, false);
    }

    default TopicStats getStats(String topic) throws PulsarAdminException {
        return getStats(topic, false, false);
    }
```

Reply:
Maybe we could return a completable future object by the new API?

Reply:
Fixed by using a new API `TopicStats getStats(GetStatsOptions options)`